app/views/challenge/gift_exchange/_challenge_signups.html.erb

```diff
@@ -13,40 +13,40 @@
   <li class="search" role="search">
     <%= form_tag collection_signups_path(@collection), class: "simple search", method: :get do |form| %>
       <fieldset>
-        <%= text_field_tag "query", @query, title: "search" %>
-        <%= submit_tag ts("Search By Pseud") %>
+        <%= text_field_tag "query", @query, title: t(".navigation.search"), autocomplete: "on" %>
+        <%= submit_tag t(".navigation.search_by_pseud") %>
       </fieldset>
     <% end %>
   </li>
   <li>
-    <%= link_to ts("Download (CSV)"), collection_signups_path(@collection, format: :csv) %> 
+    <%= link_to t(".navigation.download_csv"), collection_signups_path(@collection, format: :csv) %>
     <%= link_to_help "csv-download" %>
   </li>
 </ul>
 <!--/subnav-->
 
 <!--main content-->
 <% if @challenge_signups.empty? %>
-  <p><%= ts("No sign-ups yet!")%></p>
+  <p><%= t(".no_sign_ups_yet") %></p>
 <% else %>
   <%= will_paginate(@challenge_signups) %>
 
   <dl class="index group">
     <% @challenge_signups.each do |signup| %>
-      <dt class="participant" title="<%= ts("user name") %>">
+      <dt class="participant" title="<%= t(".particpant_username") %>">
         <%= link_to signup.pseud.name, collection_signup_path(@collection, signup) %>
         <%= mailto_link signup.pseud.user, subject: "[#{h(@collection.title)}] Message from Collection Maintainer" %>
       </dt>
       <dd>
         <%= render "challenge_signups/signup_controls", challenge_signup: signup, subnav: false %>
         <ul class="actions">
           <li>
-            <%= link_to h(ts("Requests")) + ' &#8595;'.html_safe, "#", class: "requests_#{signup.id}_open" %>
-            <%= link_to h(ts("Close Requests")) + ' &#8593;'.html_safe, "#", class: "requests_#{signup.id}_close" %>
+            <%= link_to t(".requests_html", down_arrow: "&#8595;"), "#", class: "requests_#{signup.id}_open" %>
+            <%= link_to t(".close_requests_html", up_arrow: "&#8593;"), "#", class: "requests_#{signup.id}_close" %>
           </li>
           <li>
-            <%= link_to h(ts("Offers")) + ' &#8595;'.html_safe, "#", class: "offers_#{signup.id}_open" %>
-            <%= link_to h(ts("Close Offers")) + ' &#8593;'.html_safe, "#", class: "offers_#{signup.id}_close" %>
+            <%= link_to t(".offers_html", down_arrow: "&#8595;"), "#", class: "offers_#{signup.id}_open" %>
+            <%= link_to t(".close_offers_html", up_arrow: "&#8593;"), "#", class: "offers_#{signup.id}_close" %>
```

Comment:
Could the arrows be directly in the yml file? (I can't easily test this locally to find out.)

Reply:
I didn't have a technical reason for extracting them, I will ask T if they mind the HTML code in the string itself

Reply:
T said elsewhere it's fine to keep in the translations file, so I've put it there